### ISSUE TITLE
[Request] delay function

### ISSUE THREAD

**Issue by lrmtheboss** [ClubObsidian/DynamicGui#218]
a new function such as `delay: <time in seconds>` for waiting in between functions that take awhile and/or need to finish before the next function is executed that where otherwise things would break

one example is where your running a series of commands that depend on the previous ones being run and finished

**Comment by virustotalop** [comment]
This could be done but I would need to implement asynchronous functions, I might look into this for the 5.0.0 release and see if it is viable.

**Comment by virustotalop** [comment]
Superseded by #219
